jekyll/_cci2_ja/deployment-integrations.adoc

```diff
@@ -19,7 +19,7 @@ toc::[]
 [discrete]
 == 概要
 
-ソフトウェア・アプリケーションの開発およびテストが完了したら、デプロイし、対象ユーザーが利用できるようにする必要があります。 CircleCI can deploy to virtually any target, and can be easily configured to integrate with other services for QA/testing, feature management, and deployment strategies such as blue-green or canary deployment. 完全に自動化されたプロセスが必要な場合も、手動での承認要素が必要な場合も、お客様の要件に合わせて迅速かつ簡単に設定をカスタマイズすることができます。
```

Comment:
[no action required; nothing to do with this PR]
This original 🇺🇸 text feels a little too long (imo) 😅 
It may be love to shorten it, or break it down to lists for easier reading.
https://developers.google.com/tech-writing/one/lists-and-tables#choose_the_correct_type_of_list